src/components/dashboard/main-content/course-enrollments/data/tests/hooks.test.jsx

```diff
@@ -1,31 +1,38 @@
 import { renderHook, act } from '@testing-library/react-hooks';
 import * as logger from '@edx/frontend-platform/logging';
+import { AppContext } from '@edx/frontend-platform/react';
 import camelCase from 'lodash.camelcase';
 import dayjs from 'dayjs';
 
-import { useCourseEnrollments, useCourseUpgradeData } from '../hooks';
+import {
+  useContentAssignments,
+  useCourseEnrollments,
+  useCourseEnrollmentsBySection,
+  useCourseUpgradeData,
+} from '../hooks';
 import * as service from '../service';
-import { COURSE_STATUSES } from '../constants';
+import { COURSE_STATUSES, LEARNER_ACKNOWLEDGED_ASSIGNMENT_CANCELLATION_ALERT, LEARNER_ACKNOWLEDGED_ASSIGNMENT_EXPIRATION_ALERT } from '../constants';
 import { transformCourseEnrollment } from '../utils';
 import { createRawCourseEnrollment } from '../../tests/enrollment-testutils';
 import { createEnrollWithLicenseUrl, createEnrollWithCouponCodeUrl } from '../../../../../course/data/utils';
-
-jest.mock('react', () => ({
-  ...jest.requireActual('react'),
-  useContext: jest.fn(() => ({
-    state: {
-      course: {
-        entitlements: {
-          mode: 'paid-executive-education',
-          price: '820.00',
-          currency: 'USD',
-          sku: '821D85D',
-          expires: null,
-        },
-      },
-    },
-  })),
-}));
+import { ASSIGNMENT_ACTION_TYPES, ASSIGNMENT_TYPES } from '../../../../../enterprise-user-subsidy/enterprise-offers/data/constants';
+
+// jest.mock('react', () => ({
+//   ...jest.requireActual('react'),
+//   useContext: jest.fn(() => ({
+//     state: {
+//       course: {
+//         entitlements: {
+//           mode: 'paid-executive-education',
+//           price: '820.00',
+//           currency: 'USD',
+//           sku: '821D85D',
+//           expires: null,
+//         },
+//       },
+//     },
+//   })),
+// }));
```

Comment:
[nit] remaining commented out code

Reply:
Ha! D'oh, good catch. Note to future selves, don't mock `useContext` because this would previously affect _all_ usages of `useContext` across all hooks. Commenting out the mock didn't break any existing tests; not sure why this mock is here to begin with lol. Will remove!